application-ckeditor-ui/src/main/resources/CKEditor/VelocityMacros.xml

```diff
@@ -214,12 +214,14 @@
   #end
   #set ($officeImporterSupported = $officeServerConnected &amp;&amp;
     "$!services.wysiwyg.importOfficeAttachment($missingAttachmentReference, {'useOfficeViewer': true})" != '')
+  #set ($temporaryAttachmentSupported = $services.wysiwyg.isTemporaryAttachmentSupported())
   #set ($discard = $parameters.attributes.putAll({
     'class': "$!parameters.attributes.get('class') ckeditor-textarea loading",
     'spellcheck': false,
     'data-syntax': ${services.wysiwyg.HTMLSyntax.toIdString()|'annotatedxhtml/1.0'},
     'data-sourceDocumentSyntax': $sourceDocument.syntax.toIdString(),
-    'data-officeImporterSupported': $officeImporterSupported
+    'data-officeImporterSupported': $officeImporterSupported,
+    'data-isTemporaryAttachmentSupported': true
```

Comment:
That's actually not good as the feature is not there in older version of XWiki: we need to put this value based on a check of the upload API endpoint.